pkgs/development/compilers/openjdk/darwin/default.nix

```
@@ -7,10 +7,11 @@ let
   };
 
   jdk = stdenv.mkDerivation rec {
-    name = "zulu15.28.51-ca-jdk15.0.1";
+    pname = "zulu15.28.51-ca-jdk";
+    version = "15.0.1";
 
     src = fetchurl {
-      url = "https://cdn.azul.com/zulu/bin/${name}-macosx_x64.tar.gz";
+      url = "https://cdn.azul.com/zulu/bin/${pname}-${version}-macosx_x64.tar.gz";
```

The URL is this:

```suggestion
      url = "https://cdn.azul.com/zulu/bin/${pname}${version}-macosx_x64.tar.gz";
```

Is suspect this prompted the awkward naming choice in the first place

---

Thanks.